src/ci/docker/scripts/x86_64-gnu-llvm1.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+/tmp/add_dummy_commit.sh
```

Comment:
This should only be needed for the bootstrap self tests, but the way the skips are configured, these now probably run on both llvm[1/2/3]. Anyway, it doesn't hurt to run it everywhere.